Tests/Bridge/Symmetric/KeyReaderTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * This file is part of the Global Trading Technologies Ltd crypt-bundle package.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+
+namespace Gtt\Bundle\CryptBundle\Tests\Bridge\Symmetric;
+
+use PHPUnit_Framework_TestCase as TestCase;
+use Gtt\Bundle\CryptBundle\Bridge\Symmetric\KeyReader;
+use Crypto;
+
+/**
+ * Tests for key reader
+ */
+class KeyReaderTest extends TestCase
+{
+    /**
+     * Valid encryption key
+     */
+    const TEST_KEY = '0123456789ABCDEF';
+
+    /**
+     * Path to a key file
+     *
+     * @var string
+     */
+    private $filename;
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function setUp()
+    {
+        $this->filename = tempnam(sys_get_temp_dir(), 'test');
```

Comment:
Need to check if PHP_VERSION_ID < 50400 and skip all tests, because of https://travis-ci.org/GlobalTradingTechnologies/crypt-bundle/jobs/118239582#L155-L156